core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.ts

```diff
@@ -998,6 +1008,20 @@ export class EditEmaEditorComponent implements OnInit, OnDestroy {
             },
             [CLIENT_ACTIONS.NOOP]: () => {
                 /* Do Nothing because is not the origin we are expecting */
+            },
+            [CLIENT_ACTIONS.INIT_BLOCK_EDITOR_INLINE_EDITING]: (payload) => {
+                if (!this.uveStore.isEnterprise()) {
+                    this.#dotAlertConfirmService.alert({
+                        header: this.dotMessageService.get(
+                            'dot.common.license.enterprise.only.error'
+                        ),
+                        message: this.dotMessageService.get('editpage.not.lincese.error')
+                    });
+
+                    return;
+                }
+
+                this.#dotEventsService.notify(INLINE_EDIT_BLOCK_EDITOR_EVENT, payload);
```

Comment:
Why this is not managed from the store? We can create a `computed` and a method to listen and set the current state of this sidebar 

I think is better to have that info in the store or in an feature of the store, that way we can control it if needed and we don't add another moving piece to the equation, this service

Reply:
Maybe not a new store feature, but update the editor one, to handle this property